src/views/pages/profiles/import-profile/account-selection/AccountSelectionTs.ts

```diff
@@ -175,39 +175,7 @@ export default class AccountSelectionTs extends Vue {
         if (!this.selectedAccounts.length) {
```

Comment:
this condition needs to check optIn accounts too

Reply:
I don't think so, we want to be sure that at least there's one seed account with the correct curve imported

Reply:
But user may not want to import opt in account (either if doesn't have one or if he moved the funds to another)

Reply:
@bassemmagdy I think @AdriaCarrera is right, as the opt in keys are loaded as env vars, it will always show.